docker/bash-scripts/ecr-script/docker-ecr.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+DockerRepository="";
+ProjectName="";
+Region="";
+
+echo What is the next deployment version?;
+
+read ImageVersion;
+
+echo Building and pushing version $ProjectName:$ImageVersion to ECR...;
```

Comment:
I would add a check to make sure `ImageVersion` is not empty.

Even better, another prompt asking: 

> Create build $ProjectName:$ImageVersion? (Yes/No)

that way if they enter an empty value or an incorrect one, they can confirm it instead of trying to furiously smash `Crtl+C`

Reply:
> furiously smash `Crtl+C`